From 383af0f15d1865004a03fe566a3e322e8575644a Mon Sep 17 00:00:00 2001
From: Etienne Carriere <etienne.carriere@foss.st.com>
Date: Thu, 27 Jun 2024 08:46:49 +0200
Subject: [PATCH 1115/1695] dt-bindings: mfd: dual licensing for st,stpmic1
 bindings

Change include/dt-bindings/mfd/st,stpmic1.h license model from GPLv2.0
only to dual GPLv2.0 or BSD-2-Clause. This change clarifies that
this DT binding header file can be shared with software components as
bootloaders and OSes that are not published under GPLv2 terms as
OP-TEE OS is.

This change has been discussed and acked in the LKML [1].

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 93b9370846ce39ec66f4150db1c635e7dabef45b dt-bindings: mfd: dual licensing for st,stpmic1 bindings]
Change-Id: I8818659f7f8ad304dea436f247cef1440b314789
Fixes: 1183a0aa2af0 ("stm32mp1: update DTS files to Linux kernel 5.2-rc1")
Link: https://lore.kernel.org/lkml/171941721004.2530174.778562710266249921.b4-ty@kernel.org/ [1]
(cherry picked from commit 62e40b88ef3b3e8cb27d05b9631e524d0b8eb9df)
Signed-off-by: Etienne Carriere <etienne.carriere@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/402905
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
---
 core/include/dt-bindings/mfd/st,stpmic1.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/include/dt-bindings/mfd/st,stpmic1.h b/core/include/dt-bindings/mfd/st,stpmic1.h
index 321cd0879..9dd15b9c7 100644
--- a/core/include/dt-bindings/mfd/st,stpmic1.h
+++ b/core/include/dt-bindings/mfd/st,stpmic1.h
@@ -1,4 +1,4 @@
-/* SPDX-License-Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0-only OR BSD-2-Clause */
 /*
  * Copyright (C) STMicroelectronics 2018 - All Rights Reserved
  * Author: Philippe Peurichard <philippe.peurichard@st.com>,
-- 
2.47.2

